packages/apollo/src/commands/service/__tests__/check.test.ts

```diff
@@ -1,52 +1,159 @@
-import { formatMarkdown } from "../check";
+import {
+  formatHumanReadable,
+  formatMarkdown,
+  formatTimePeriod
+} from "../check";
 import checkSchemaResult from "./fixtures/check-schema-result.json";
 import { ChangeType } from "apollo-language-server/lib/graphqlTypes";
 
-describe("markdown formatting", () => {
-  it("is correct with breaking changes", () => {
-    expect(
-      formatMarkdown({
-        serviceName: "engine",
-        tag: "staging",
-        checkSchemaResult
-      })
-    ).toMatchInlineSnapshot(`
+describe("service:check", () => {
+  describe("markdown formatting", () => {
+    it("is correct with breaking changes", () => {
+      expect(
+        formatMarkdown({
+          serviceName: "engine",
+          tag: "staging",
+          checkSchemaResult
+        })
+      ).toMatchInlineSnapshot(`
 "
 ### Apollo Service Check
 🔄 Validated your local schema against schema tag 'staging' on service 'engine'.
-🔢 Compared **18 schema changes** against operations seen over the **last day**.
+🔢 Compared **18 schema changes** against operations seen over the **last 24 hours**.
 ❌ Found **7 breaking changes** that would affect **3 operations**
 
 🔗 [View your service check details](https://engine-dev.apollographql.com/service/engine/checks?schemaTag=Detached%3A%20d664f715645c5f0bb5ad4f2260cd6cb8d19bbc68&schemaTagId=f9f68e7e-1b5f-4eab-a3da-1fd8cd681111&from=2019-03-26T22%3A25%3A12.887Z).
 "
 `);
-  });
+    });
 
-  it("is correct with no breaking changes", () => {
-    expect(
-      formatMarkdown({
-        serviceName: "engine",
-        tag: "staging",
-        checkSchemaResult: {
-          ...checkSchemaResult,
-          diffToPrevious: {
-            ...checkSchemaResult.diffToPrevious,
-            type: ChangeType.NOTICE,
-            affectedQueries: [],
-            changes: []
+    it("is correct with no breaking changes", () => {
+      expect(
+        formatMarkdown({
+          serviceName: "engine",
+          tag: "staging",
+          checkSchemaResult: {
+            ...checkSchemaResult,
+            diffToPrevious: {
+              ...checkSchemaResult.diffToPrevious,
+              type: ChangeType.NOTICE,
+              affectedQueries: [],
+              changes: []
+            }
           }
-        }
-      })
-    ).toMatchInlineSnapshot(`
+        })
+      ).toMatchInlineSnapshot(`
 "
 ### Apollo Service Check
 🔄 Validated your local schema against schema tag 'staging' on service 'engine'.
-🔢 Compared **0 schema changes** against operations seen over the **last day**.
+🔢 Compared **0 schema changes** against operations seen over the **last 24 hours**.
 ✅ Found **no breaking changes**.
 
 🔗 [View your service check details](https://engine-dev.apollographql.com/service/engine/checks?schemaTag=Detached%3A%20d664f715645c5f0bb5ad4f2260cd6cb8d19bbc68&schemaTagId=f9f68e7e-1b5f-4eab-a3da-1fd8cd681111&from=2019-03-26T22%3A25%3A12.887Z).
 "
 `);
+    });
+  });
+
+  describe("formatTimePeriod", () => {
```

Comment:
Great advice. The 1 hour code is wrong :)

Reply:
I actually think it _wasn't_ wrong, but the change you made is also not incorrect. I only asked for the test since I could see the edge case in the function, but it made sense in the way that you would use it.

"xxx in the last hour"
"xxx in the last 24 hours"

So I think you've changed it to
"xxx in the last 1 hour"

Either way seems fine to me! 👍

Reply:
Good eye 😃 You're right!